packages/cli/src/commands/runAndroid/__tests__/sampleBuild.gradle

```diff
@@ -0,0 +1,150 @@
+apply plugin: "com.android.application"
```

Comment:
we usually store such files inside `__fixtures__` directory next to `__tests__`, can you move it there? :)

Reply:
No problem.